### PR TITLE
Fix the default executor host since it's changed in 1.4

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

Since the executor's hostname is now `executor`

https://github.com/appwrite/appwrite/blob/36698d180c44c631576607569f4255afc2ead7ca/app/views/install/compose.phtml#L592-L595

the default variable value should be updated.

## Test Plan

Manual

## Related PRs and Issues

None

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Comment by abnegate** [comment]
I think we'll need to change the Dockerfile default as well

**Comment by stnguyen90** [comment]
@abnegate, the env var in the Dockerfile shouldn't matter after https://github.com/appwrite/appwrite/pull/5971

File: app/config/variables.php [hunk 1]
```diff
@@ -748,7 +748,7 @@
                 'name' => '_APP_EXECUTOR_HOST',
                 'description' => 'The host used by Appwrite to communicate with the function executor!',
                 'introduction' => '0.13.0',
-                'default' => 'http://appwrite-executor/v1',
+                'default' => 'http://executor/v1',
```
Comment (Meldiron) [inline app/config/variables.php:751]: Looks good, but maybe we should go back to using `appwrite-executor`, or have smart migration script (if value is exactly `http://appwrite-executor/v1`, then replace. If not, keep using their modified value)

Comment (stnguyen90) [reply app/config/variables.php:751]: let's leave it as executor. smarter installation command coming in: https://github.com/appwrite/appwrite/issues/6137

Comment (christyjacob4) [reply app/config/variables.php:751]: @stnguyen90 the override PR has been merged. I think now we should change it back to `appwrite-executor`

**Pull request closed without merging**